cmd/agentless-scanner/aws-deploy.sh

```diff
@@ -0,0 +1,263 @@
+#!/bin/bash
```

Comment:
Can you update the codeowner for files in `cmd/agentless-scanner`?

Reply:
Done. Thanks.